git-changelog.gemspec

```diff
@@ -6,10 +6,11 @@ Gem::Specification.new do |s|
   s.description = "Write your changelog as you go into your commit messages. This tool generates a useful changelog from marked lines in your git commit messages"
   s.authors     = ["Markus Chmelar"]
   s.email       = 'markus.chmelar@innovaptor.com'
-  s.files       = ["bin/git-changelog","lib/changelog_helpers.rb","lib/git-changelog.rb"]
+  s.files       = ["bin/git-changelog","lib/changelog_helpers.rb","lib/git-changelog.rb", "lib/changelog.rb"]
   s.homepage    = 'https://github.com/iv-mexx/git-changelog'
   s.license     = 'MIT'
   s.executables = "git-changelog"
   s.add_runtime_dependency 'docopt', '~> 0.5', '>= 0.5.0'
   s.add_runtime_dependency 'rugged'
```

Comment:
I think we should add `>= 0.23.0` for rugged. The reason for that being in the Gemfile was, that at that time, 0.23 was not yet released and I did not know how specify a git-tag in the gemspec, hence the entry in the Gemfile.